theory/tests/test_periodic.c

```diff
@@ -707,7 +707,7 @@ int main(int argc, char **argv)
                 total_tests++;
                 read_data_and_set_globals(firstfilename[this_test_num],firstfiletype[this_test_num],secondfilename[this_test_num],secondfiletype[this_test_num]);
                 pimax=allpimax[this_test_num];
-                options.boxsize = allboxsize[i];
+                options.boxsize = allboxsize[this_test_num];
```

Comment:
Huh - this looks like a separate bugfix. Is that the case?

Reply:
Yep! I discovered it while debugging.